packages/apidom-ls/src/config/common/schema/lint/additional-items--type.ts

```diff
@@ -9,7 +9,7 @@ const additionalItemsTypeLint: LinterMeta = {
   message: 'additionalItems must be a schema',
   severity: DiagnosticSeverity.Error,
   linterFunction: 'apilintElementOrClass',
-  linterParams: ['schema'],
+  linterParams: ['schema', 'boolean'],
```

Comment:
We cannot really do that. OpenAPI 2.0/3.0.x use JSON Schema Draft 4/5 which doesn't support `Boolean JSON Schema`.  Boolean JSON Schemas are support since JSON Schema Draft 6 and applies to: AsyncAPI 2.x and OpenAPI 3.1.0 only.

We would have to fork this rule and use `targetSpec` to apply it to appropriate specification and their versions.

> [!IMPORTANT]  
> This applies to all modified rules in `/common/schema/`